eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -143,7 +143,7 @@ stages:
               Cloud: ${{ cloud.key }}
 
 - ${{ if contains(variables['Build.DefinitionName'], 'tests-weekly')  }}:
-  - stage: Analysis
+  - stage: ${{ parameters.JobName }}_Analysis
     dependsOn: []
```

Comment:
Another option is to remove the stage name here and let devops give it an id and we can still set the displayName which can be set to whatever we want, including the same name if desired.

Reply:
I'm going to go this route. Thanks for the suggestion Wes.

Reply:
With the display name you also don't have to use things like '_' if you don't want as spaces are allowed.